cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportContext.java

```diff
@@ -194,41 +140,14 @@ public CgmesExportContext(Network network, ReferenceDataProvider referenceDataPr
             topologyKind = networkTopologyKind(network);
         }
         scenarioTime = network.getCaseDate();
-        CgmesMetadataModels models = network.getExtension(CgmesMetadataModels.class);
-        if (models != null) {
-            models.getModelForSubset(CgmesSubset.EQUIPMENT).ifPresent(eq -> prepareExportedModelFrom(exportedEQModel, eq));
-            models.getModelForSubset(CgmesSubset.STEADY_STATE_HYPOTHESIS).ifPresent(ssh -> prepareExportedModelFrom(exportedSSHModel, ssh));
-            models.getModelForSubset(CgmesSubset.TOPOLOGY).ifPresent(tp -> prepareExportedModelFrom(exportedTPModel, tp));
-            models.getModelForSubset(CgmesSubset.STATE_VARIABLES).ifPresent(sv -> prepareExportedModelFrom(exportedSVModel, sv));
-        }
         addIidmMappings(network);
-    }
 
-    /**
-     * Set for each exported model the profile relative to the cim version.
-     * @param cim The cim version from which depends the models profiles uri.
-     */
-    private void initializeExportedModelProfiles(CgmesNamespace.Cim cim) {
-        if (cim.hasProfiles()) {
-            exportedEQModel.setProfile(cim.getProfileUri("EQ"));
-            exportedTPModel.setProfile(cim.getProfileUri("TP"));
-            exportedSVModel.setProfile(cim.getProfileUri("SV"));
-            exportedSSHModel.setProfile(cim.getProfileUri("SSH"));
+        if (network.hasProperty(BOUNDARY_EQ_ID_PROPERTY)) {
+            setBoundaryEqId(network.getProperty(BOUNDARY_EQ_ID_PROPERTY));
+        }
+        if (network.hasProperty(BOUNDARY_TP_ID_PROPERTY)) {
+            setBoundaryTpId(network.getProperty(BOUNDARY_TP_ID_PROPERTY));
         }
-    }
-
-    /**
-     * Update the model used for the export according to the network model.
-     * All the metadata information will be duplicated, except for the version that will be incremented.
-     * @param exportedModel The {@link CgmesMetadataModel} used for the export.
-     * @param fromModel The {@link CgmesMetadataModel} attached to the network.
-     */
-    private void prepareExportedModelFrom(CgmesMetadataModel exportedModel, CgmesMetadataModel fromModel) {
-        exportedModel.setDescription(fromModel.getDescription());
-        exportedModel.setVersion(fromModel.getVersion() + 1);
-        exportedModel.addSupersedes(fromModel.getId());
```

Comment:
The md:Model.Supersedes is no longer exported in the SSH and SV file

Reply:
Same comment as above. Please review the added unit test. The supersedes information must also be obtained from the output file.